frameworks/shared/requirements.in

```diff
@@ -1,5 +1,4 @@
 psutil>=5.4
-ruamel.yaml>=0.15
```

Comment:
as I was rebuilding the dependencies, I also refactored imports in `amlb.utils` to be able to remove this one from frameworks' requirements.